DEVELOPMENT.md

```diff
@@ -181,10 +181,16 @@ For example, if you want to run all of the Exact Filter Benchmarks, you could ru
 ./hack/run.sh benchmark-filter ExactFilterBenchmark
 ```
 
+To run benchmarks for any filter benchmark , you can use:
+
+```shell
+./hack/run.sh benchmark-filter AnyFilterBenchmark
+```
+
 Alternatively, if you want to run all of the benchmarks you can run:
 
 ```shell
-./hack/run.sh benchmark-filters
+./hack/run.sh benchmark-filter AllFilterBenchmark
 ```
```

Comment:
I don't think this change makes sense, as we are already pointing users to the list of available filters on line 177. The lines you deleted were actually showing that you can call `./hack/run.sh benchmark-filters`, which will benchmark all of the filters, not just one of them.

Would you be able to revert the changes to this file?